docs/users.md

```diff
@@ -0,0 +1,41 @@
+We track different types of users in our browser code.
+
+There are three disjoint sets of users you care about
+for typical Zulip realms:
+
+- active users in your realm
+- cross-realm users like feedback@zulip.com
+- deactivated users
+
+You can also think in terms of these user populations:
+
+- current realm users (active users in your realm)
+- active users (adds cross-realm bots to current realm users)
+- all users (adds deactivated users to active users)
+
+Each of the above categories makes sense for certain UI functions.
+
+Let's start with the features that are restricted to active realm
+users, and let's end with features that can apply to all users.
+
+Only current realm users can...
+- be subscribed to public streams on your realm
+- be highlighted in the compose-fade feature
+- show up in your buddy list
+
+Active users can...
+- be sent PMs to
+- show up in your compose recipient typeaheads
+- show up in your at-mention typeaheads
```

Comment:
I think only current realm users should be in @-mention typeaheads -- logically, @-mentioning feedback-bot in a stream won't work.